### ISSUE TITLE
mrtk style ui tools

### ISSUE THREAD

**Issue by peterclemenko** [GodotVR/godot-xr-tools#681]
integrating mrtk style ui tools into xr tools would be a godsend for xr ux development

**Comment by Malcolmnixon** [comment]
It's certainly doable, although it would be best to implement this as a separate addon - Godot-XR-Tools is already too large and in the future we will be splitting the functionality into separate addons.

**Comment by BastiaanOlij** [comment]
Indeed, this needs further discussion. I think at some point a proper 3D interactable UI would be nice to have but whether this should be part of XR tools itself or a separate plugin. IDK.
